OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -21,9 +21,34 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	[Flags]
+	public enum FlightDynamic
+	{
+		None = 0,
+		MoveIntoShroud = 1,
+		Slide = 2,
+		Hover = 4,
+		VTOL = 8,
+		VTOLTurnToLand = 16,
+		VTOLTurnToDock = 32,
```

Comment:
Although this is currently not implemented for non-VTOL craft it will likely be in the future similarly to the above.